gridcomps/Cap/MAPL_CapGridComp.F90

```diff
@@ -137,8 +140,16 @@ subroutine MAPL_CapGridCompCreate(cap, root_set_services, cap_rc, name, final_fi
 
     meta => null()
     call MAPL_InternalStateCreate(cap%gc, meta, __RC__)
+
+    meta%t_profiler = DistributedProfiler(trim(cap_name), MpiTimerGauge(), comm=comm_world)
+    meta%user_setservices_wrapper = ProcSetServicesWrapper(set_services_gc)
     call MAPL_Set(meta, CF=cap%config, __RC__)
 
+    block
+      character(ESMF_MAXSTR) :: root_name
```

Comment:
I cannot see where we use the variable 'root_name', besides getting it from a resource file

Reply:
OK - I'll take a look. Presumably it is used to give the name of the ROOT GC.

Reply:
Must have been something "lost" when moving chunks of code around.  Definitely not needed here - as it is doing nothing.
  I will fix.

Reply:
This is what I thought too, but I did not see it (plus the variable scope is limited to the block)